src/compiler/diagnosticMessages.json

```diff
@@ -2989,6 +2989,14 @@
         "category": "Message",
         "code": 6148
     },
+    "Did you mean to import the default export, with no curly braces":{

```

Comment:
how about:

`Module '{0}' has no default export. Did you mean to use 'import {{1}} from "{0}"'instead?`